src/drc/api/serializers.py

```diff
@@ -331,6 +452,37 @@ def validate(self, attrs):
         return valid_attrs
 
 
+class EnkelvoudigInformatieObjectCreateLockSerializer(EnkelvoudigInformatieObjectSerializer):
```

Comment:
does this correctly show up in the generated schema?

Reply:
after removing request-related condition from `get_serializer` the schema is generated correctly